### PR TITLE
Bug 1822349 - update migration data for firefox-android

### PR DESCRIPTION
**Pull request by jcristau**
### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1822349

### CODE REVIEW

**Review by gbrownmozilla — APPROVED** [review]
I'm not very familiar with the monorepo-migration contents, but this looks right to me.

File: monorepo-migration/generate-replace-message-expressions.py [hunk 1]
```diff
@@ -35,9 +35,7 @@ def main():
     with open(DATA_DIR / "repo-numbers.json") as f:
         repo_numbers = json.load(f)
 
-    regexes = [
-        "regex:^==>[fenix] ",
```
Comment (JohanLorenzo) [inline monorepo-migration/generate-replace-message-expressions.py:39]: Okay to stop prefixing commit messages with `[REPO]`

File: monorepo-migration/data/repo-numbers.json [hunk 2]
```diff
@@ -41603,7 +41591,1823 @@
...
-            28830
+            28830,
+            28831,
+            28832,
+            28833
+        ]
+    },
+    "firefox-android": {
+        "issues": [],
+        "pulls": [
```
Comment (JohanLorenzo) [inline monorepo-migration/data/repo-numbers.json:41602]: These numbers LGTM 👌

File: monorepo-migration/data/repo-numbers.json [hunk 3]
```diff
@@ -30202,7 +30191,6 @@
             28142,
             28145,
             28150,
-            28152,
```
Comment (JohanLorenzo) [inline monorepo-migration/data/repo-numbers.json:30205]: It's weird these PRs/issues disappeared from Github. E.g.: https://github.com/mozilla-mobile/fenix/issues/28152

That said, there's nothing we can do in this current PR.

**Review by JohanLorenzo — APPROVED** [review]
LGTM!

**Pull request merged**